application/controllers/admin/bestitamazonpay4oxid_init.php

```diff
@@ -306,9 +306,9 @@ public static function getCurrentVersion()
         );
 
         foreach ($aPossibleModuleNames as $sPossibleModuleName) {
-            if ($aVersions[$sPossibleModuleName]) {
+            if (isset($aVersions[$sPossibleModuleName]) && $aVersions[$sPossibleModuleName]) {
```

Comment:
I think the `&& $aVersions[$sPossibleModuleName]` isn't necessary in both cases.

Reply:
It depends what you want to check here: only if the key in array exists, or also if the array entry is not empty? In first case array_keys_exists would be also enough.

Reply:
Just remove the `&& $aVersions[$sPossibleModuleName]` and I will merge it. In this case we need a valid value, so `null` isn't that what we want and `array_key_exists` is a quiet bad function for big arrays from a performance based view.